dbms/src/Storages/Page/V3/PageEntriesEdit.h

```diff
@@ -90,6 +90,7 @@ enum class EditRecordType
     VAR_REF,
     VAR_EXTERNAL,
     VAR_DELETE,
+    UPDATE_REMOTE,
```

Comment:
For `UPDATE_REMOTE`, do we need to distinguish between snapshot types and in-memory types?

Reply:
Don't get what is "snapshot types" and "in-memory types".

Reply:
It won't reside in `PageDirectory`, it is just used to update the local cache info for `VAR_ENTRY` type.

Reply:
Then maybe we should put it after UPSERT

Reply:
The order of `EditRecordType` can't be changed. Because the order determines the underlying value for each enum value which will be written to the WAL.
Just add some comments in it to distinguish the new type between `VAR_` types.

Reply:
OK